### PR TITLE
[GOV] New `aeon` governance document.

### PR DESCRIPTION
**Pull request by MatthewMiddlehurst**
This PR contains changes to the governance document in the style on `Scikit-learn`, based off my opinions from #2. All of this is flexible, but I think we should cut down the size of the original document whatever we go with.

This is a full rewrite of the governance document. Diff can be seen here but its not very helpful https://gist.github.com/MatthewMiddlehurst/b51db154402ecd88ca7587133b65e28a/revisions

Key changes:
- Removes algorithm maintainer role
- Removes CC observer role
- Removes some CC power i.e. veto right

current gov page: https://www.aeon-toolkit.org/en/latest/get_involved/governance.html
new gov page: https://aeon-toolkit--195.org.readthedocs.build/en/195/get_involved/governance.html

### CODE REVIEW

File: docs/source/get_involved/governance.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+### Contributors
+
+Contributors are community members who have contributed in concrete ways to the project. Anyone can become a contributor, and contributions can take many forms – not only code – as detailed in the contributing guide.
+
+### Core Developers
+
+Core developers are community members that have made non-trivial contributions and are trusted in continuing the development of the project and engaging with its community. Core Developers are granted write access to the ``aeon`` repository and voting rights on most project decisions. Core Developers are expected to review code contributions and engage with topics or code they are knowledgeable about.
+
+New Core Developers are nominated by existing Core Developers, and are subject to a two-third majority vote of existing Core Developers. Core Developers are expected to maintain a reasonable amount of engagement with the project, and may lose their status after a long period of inactivity and lack of engagement with fellow developers. Developing code, interacting with contributions and engaging with the broader community are all valid contributions for Core Developers.
```
Comment (TonyBagnall) [inline docs/source/get_involved/governance.md:19]: I probably wouldn't put the bit about "losing status" at all, and may not have the 2/3 bit. I would prefer "New Core Developers can be nominated by any contributor (including themselves) and are appointed with the agreement of the current Core Developers" and scrap this whole paragraph

File: docs/source/get_involved/governance.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+### Community Council
+
+Community Council (CC) members are Core Developers who have additional responsibilities to ensure the smooth running of the project. This includes strategic planning, project infrastructure management and release management. In the event Core Developers fail to reach consensus, the CC is the entity to resolve the issue.
+
+CC membership is subject to a two-third majority vote of Core Developers. CC terms last for two years, after which a new vote must be held.
+
+### Code of Conduct Committee
+
+Code of Conduct Committee (CoCC) members are contributors which are tasked with maintaining the ``aeon`` Code of Conduct (CoC) and managing CoC reports. CoC members are expected to review reports of CoC violations, contact and discuss with involved individuals and make recommendations on actions to take.
```
Comment (TonyBagnall) [inline docs/source/get_involved/governance.md:29]: managing CoC reports -> " managing reports of breaking the CoC."

File: docs/source/get_involved/governance.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+CoCC membership is subject to a two-third majority vote of Core Developers. CoCC terms last for two years, after which a new vote must be held. CC members cannot simultaneously be CoCC members.
+
+## Decision-Making Process
+
+Decisions about the future of the project are made publicly to allow discussion with all members of the community. All non-sensitive project management discussion takes place on the contributors channel on the project Slack and/or the issue tracker. Occasionally, sensitive discussion and votes such as appointments will occur in private Slack channels.
+
+For most decisions a consensus seeking process of all interested contributions is used. Contributors try to find a resolution that has no open objections among Core Developers. If a reasonable amount of time has passed since the last change to a proposed contribution, the proposal has at least one approval (+1) and no rejections (-1) from Core Developers, it can be approved by lazy consensus. If a change is rejected, it is expected that some form of explanation and description of conditions (if any) to withdraw the rejection is provided.
```
Comment (TonyBagnall) [inline docs/source/get_involved/governance.md:39]: interested contributions -> interested contributors

Comment (TonyBagnall) [inline docs/source/get_involved/governance.md:39]: lazy consensus means no one has approved it. I say remove the concept all together. Lack of interest in a PR is to me a sign that its not really needed or of interest

File: docs/source/get_involved/governance.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+At any point during the discussion, any Core Developer in favour of a change can call for a vote, which will conclude two weeks from the call for the vote. Any vote must be backed by an AEP (see following section). If no option can gather two-thirds of the votes cast, the decision is escalated to the Community Council, which in turn will use consensus seeking with the fallback option of a simple majority vote if no consensus can be found.
+
+All changes to the ``aeon`` code or documentation should be done via Pull Request. By default, push rights to the ``main`` GitHub branch are restricted for all developers. In emergencies, the Community Council may temporarily revoke the branch protection for council members and make direct commits, but this should happen sparingly and only in circumstances where harm will come to the project unless timely action is taken.
```
Comment (TonyBagnall) [inline docs/source/get_involved/governance.md:43]: "happen only in emergencies where harm will come to the project unless timely action is taken"

**Review by TonyBagnall — CHANGES_REQUESTED** [review]

File: docs/source/get_involved/governance.md (hunk 3 above)
Comment (MatthewMiddlehurst) [reply docs/source/get_involved/governance.md:39]: Scikit-learn uses it in a different context at least, the line is similar to what they have. Can remove it as it doesn't really change the meaning, though.

File: docs/source/get_involved/governance.md (hunk 1 above)
Comment (MatthewMiddlehurst) [reply docs/source/get_involved/governance.md:19]: Think we should keep the requirements to join there, but the exact thresholds and who can nominate are up for debate as you suggest.

I think we should be upfront about expectations (if there are any and whatever they are). Having nothing would kind of imply as long as you get voted in you remain forever.

**Comment by TonyBagnall** [comment]
an alternative for discussion

File: docs/source/get_involved/governance.md (hunk 3 above)
Comment (TonyBagnall) [reply docs/source/get_involved/governance.md]: I just dont like the term "lazy consensus" personally, otheres might like it.

**Comment by TonyBagnall** [comment]
chatgpt likes it. Here is what is says about the rest

The text is generally well written and clear. However, here are some suggestions to improve it:

Governance and decision-making
The purpose of this document is to formalize the governance process used by the aeon project, to clarify how decisions are made and how the various elements of our community interact. Our goal is to ensure a transparent, democratic, and inclusive decision-making process that empowers all community members to contribute to the project.

Aeon is a community-owned and community-run project. Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision-making process. We believe that diversity and inclusivity are critical to the success of the project and welcome contributions from all backgrounds and perspectives.

This document establishes the decision-making structure used by aeon, which strives to find consensus from the community while avoiding any deadlocks. Our decision-making process involves different roles, each with specific responsibilities.

Roles
Contributors
Contributors are community members who have contributed in concrete ways to the project. Anyone can become a contributor, and contributions can take many forms – not only code – as detailed in the contributing guide. Contributors play a crucial role in shaping the project and can participate in discussions and decision-making processes.

Core Developers
Core developers are community members who have made significant contributions and are trusted to continue the development of the project and engage with its community. Core developers are granted write access to the aeon repository and voting rights on most project decisions. They are expected to review code contributions and engage with topics or code they are knowledgeable about.

New core developers are nominated by existing core developers and are subject to a two-thirds majority vote of existing core developers. Core developers are expected to maintain a reasonable amount of engagement with the project and may lose their status after a long period of inactivity and lack of engagement with fellow developers. Developing code, interacting with contributions, and engaging with the broader community are all valid contributions for core developers.

Community Council
The Community Council (CC) consists of core developers who have additional responsibilities to ensure the smooth running of the project. This includes strategic planning, project infrastructure management, and release management. In the event that core developers fail to reach a consensus, the CC is responsible for resolving the issue.

CC membership is subject to a two-thirds majority vote of core developers, and terms last for two years, after which a new vote must be held. The CC is accountable to the community and must report regularly on its activities and decisions.

Code of Conduct Committee
The Code of Conduct Committee (CoCC) consists of contributors tasked with maintaining the aeon Code of Conduct (CoC) and managing reports of breaking the CoC. CoCC members are expected to review reports of CoC violations, contact and discuss with involved individuals, and make recommendations on actions to take.

Any CoCC members involved in a CoC report or CoCC members who have a conflict of interest regarding the report are expected to recuse themselves from the process. If no CoCC members are available to review a report in a timely manner, the responsibility will fall to the Community Council.

CoCC membership is subject to a two-thirds majority vote of core developers, and terms last for two years, after which a new vote must be held. CC members cannot simultaneously be CoCC members to avoid conflicts of interest.

File: docs/source/get_involved/governance.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+Any CoCC members involved in a CoC report or CoCC members which have a conflict of interest regarding the report are expected to recuse themselves from the process. If no CoCC members are available to review a report in a timely manner, the responsibility will fall to the Community Council.
+
+CoCC membership is subject to a two-third majority vote of Core Developers. CoCC terms last for two years, after which a new vote must be held. CC members cannot simultaneously be CoCC members.
+
+## Decision-Making Process
+
+Decisions about the future of the project are made publicly to allow discussion with all members of the community. All non-sensitive project management discussion takes place on the contributors channel on the project Slack and/or the issue tracker. Occasionally, sensitive discussion and votes such as appointments will occur in private Slack channels.
```
Comment (aiwalter) [inline docs/source/get_involved/governance.md:37]: > Decisions about the future of the project are made publicly

this is not clear to me, does it mean decision hae to be made involving public or just comunication of decision? I prefer the later here. We can still always have public discussions and sometimes also decisions if we want, but it should not be a requirement imho

File: docs/source/get_involved/governance.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+For most decisions a consensus seeking process of all interested contributors is used. Contributors try to find a resolution that has no open objections among Core Developers. If a reasonable amount of time has passed since the last change to a proposed contribution, the proposal has at least one approval (+1) and no rejections (-1) from Core Developers, it can be approved by lazy consensus. If a change is rejected, it is expected that some form of explanation and description of conditions (if any) to withdraw the rejection is provided.
```
Comment (aiwalter) [inline docs/source/get_involved/governance.md:39]: > reasonable amount of time

I would propose to add 7 days here to be clear

File: docs/source/get_involved/governance.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+At any point during the discussion, any Core Developer in favour of a change can call for a vote, which will conclude two weeks from the call for the vote. Any vote must be backed by an AEP (see following section). If no option can gather two-thirds of the votes cast, the decision is escalated to the Community Council, which in turn will use consensus seeking with the fallback option of a simple majority vote if no consensus can be found.
```
Comment (aiwalter) [inline docs/source/get_involved/governance.md:41]:
```suggestion
At any point during the discussion, any Core Developer in favour of a change can call for a vote, which will conclude one weeks from the call for the vote. Any vote must be backed by an AEP (see following section) or an issue or PR describing the objective of the vote. If no option can gather two-thirds of the votes cast, the decision is escalated to the Community Council, which in turn will use consensus seeking with the fallback option of a simple majority vote if no consensus can be found.
```

File: docs/source/get_involved/governance.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+All changes to the ``aeon`` code or documentation should be done via Pull Request. By default, push rights to the ``main`` GitHub branch are restricted for all developers. In emergencies, the Community Council may temporarily revoke the branch protection for council members and make direct commits, but this should happen only in emergencies where harm will come to the project unless timely action is taken.
```
Comment (aiwalter) [inline docs/source/get_involved/governance.md:43]: > developers

is it a Core Developer?

File: docs/source/get_involved/governance.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+### Enhancement Proposals
+
+For all non-appointment decision-making votes, a proposal must have been made public and discussed before the vote. Such proposal must be a consolidated document, in the form of a “aeon Enhancement Proposal” (AEP). Having a rejection on a pull request does not necessitate the creation of an AEP and further discussion to find consensus can be held, but one must be created prior to any vote.
```
Comment (aiwalter) [inline docs/source/get_involved/governance.md:47]: I think we need clearer description when an AEP has to be done. I thought that we had this better before? I think it shoukd mostly be optional to lower the entry point to make a contribution. AEP can be ok for major feature requests or interface changes. Basically I would say if 2/3 of CC says an AEP should be done for an enhancement then it has to be done, otherwise optional. What you think?

**Review by aiwalter — COMMENTED** [review]
left some comments. I think we need to define also when a core developer role ends. Would like to avoid having people that do not contribute but want to make decisions.

File: docs/source/get_involved/governance.md (hunk 5 above)
Comment (MatthewMiddlehurst) [reply docs/source/get_involved/governance.md:37]: Depends on the decision, really. Most decisions will be core-developer votes, but the fact that these votes are happening should be public to allow people to comment or involve themselves.

File: docs/source/get_involved/governance.md (hunk 6 above)
Comment (MatthewMiddlehurst) [reply docs/source/get_involved/governance.md:39]: I would think it depends on the change, for small bugfixes or typos we probably don't need 7 days, but we can clarify that non-trivial changes should try to give at least 7 days.

File: docs/source/get_involved/governance.md (hunk 8 above)
Comment (MatthewMiddlehurst) [reply docs/source/get_involved/governance.md:43]: Don't think its role specific, no one should be pushing to main unless it's an emergency.

File: docs/source/get_involved/governance.md (hunk 9 above)
Comment (MatthewMiddlehurst) [reply docs/source/get_involved/governance.md:47]: Ideally we should be avoiding votes, and any PR where a vote is required will be somewhat controversial if people are unable to reach a consensus through discussion. Given that the onus is on the person making the pull request to call a vote to try to push their contribution through, it should probably be accompanied by some sort of proposal.

What this proposal is and how detailed it has to be is up for debate really. @TonyBagnall did not seem so keen on having a templated "AEP". Having a reasonably detailed issue or PR description should be fine.

File: docs/source/get_involved/governance.md (hunk 5 above)
Comment (MatthewMiddlehurst) [reply docs/source/get_involved/governance.md]: "Decisions about the future of the project are announced publicly to allow discussion with all members of the community, ideally with the whole process from proposal to implementation remaining fully visible, barring sensitive topics."

File: docs/source/get_involved/governance.md (hunk 6 above)
Comment (MatthewMiddlehurst) [reply docs/source/get_involved/governance.md]: "If a reasonable amount of time (at least 7-days for non-trivial changes) has passed since the last change to a proposed contribution"

File: docs/source/get_involved/governance.md (hunk 9 above)
Comment (TonyBagnall) [reply docs/source/get_involved/governance.md]: I dont mind really, not my kind of thing but others may like the formal AEP

File: docs/get_involved/governance.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+
+Contributors are community members who have contributed in concrete ways to the project.
+Anyone can become a contributor, and contributions can take many forms – not only code –
+as detailed in the contributing guide.
+
+### Core Developers
+
+Core developers are community members that have made non-trivial contributions and are
+trusted in continuing the development of the project and engaging with its community.
+Core Developers are granted write access to the `aeon` repository and voting rights
```
Comment (TonyBagnall) [inline docs/get_involved/governance.md:26]: Developers -> developers here and later

File: docs/get_involved/governance.md [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+on most project decisions. Core Developers are expected to review code contributions
+and engage with topics or code they are knowledgeable about.
+
+New Core Developers are nominated by existing Core Developers, and are subject to a
```
Comment (TonyBagnall) [inline docs/get_involved/governance.md:30]: core developers

File: docs/get_involved/governance.md [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+
+CoCC membership is subject to a two-third majority vote of Core Developers. CoCC terms
+last for two years, after which a new vote must be held. CC members cannot
+simultaneously be CoCC members.
+
+## Decision-Making Process
+
+Decisions about the future of the project are announced publicly to allow discussion
+with all members of the community, ideally with the whole process from proposal to
+implementation remaining fully visible, barring sensitive topics. All non-sensitive
```
Comment (TonyBagnall) [inline docs/get_involved/governance.md:67]: I would reword this 

"Decisions about the future of the project are announced publicly to allow discussion
with all members of the community. The whole process from proposal to
implementation is fully visible, apart from topics considered sensitive. All non-sensitive
project management discussion takes place on the contributors channel on the project
Slack and/or the issue tracker. Occasionally, sensitive discussion and votes such as
appointments will occur in private Slack channels.

**Review by TonyBagnall — DISMISSED** [review]
I'm fine with this

File: docs/get_involved/governance.md [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+involved individuals and make recommendations on actions to take.
+
+Any CoCC members involved in a CoC report or CoCC members which have a conflict of
+interest regarding the report are expected to recuse themselves from the process.
+If no CoCC members are available to review a report in a timely manner, the
+responsibility will fall to the Community Council.
+
+CoCC membership is subject to a two-third majority vote of Core Developers. CoCC terms
+last for two years, after which a new vote must be held. CC members cannot
+simultaneously be CoCC members.
```
Comment (ltsaprounis) [inline docs/get_involved/governance.md:61]: I want to propose removing these 2 committees/councils and replace them with "Workgroups" like pandas: 
https://pandas.pydata.org/about/team.html

What do you all think?

Comment (MatthewMiddlehurst) [reply docs/get_involved/governance.md:61]: I'm not opposed to this. Would the current groups be CoC/Infrastructure/Communications/Release?

How would someone be included into a group, 50% core dev vote for regular members, 66% for lead? Do term limits still apply for both?

If this were the case, would each workgroup retain full control of the related accounts? i.e. infrastructure github/pypi/readthedocs and comms twitter/linkedin/slack?

Comment (ltsaprounis) [reply docs/get_involved/governance.md:61]: I think the control over the accounts makes sense for the workgroups. For the appointment, a vote is a good option for the future. For now I think we can aim to all agree on who does what.

Comment (MatthewMiddlehurst) [reply docs/get_involved/governance.md]: "If no option can gather two-thirds of the votes cast, the decision is escalated to the CC, which in turn will use consensus seeking with the fallback option of a simple majority vote if no consensus can be found."

Not quite sure what to do with this. Do all votes just become two-thirds or simple majority? Or would it depend on type/contents?

**Comment by MatthewMiddlehurst** [comment]
I have incorporated suggestions and made a PR into this branch for the workgroups suggestion by @ltsaprounis in #389. I suppose we can discuss that either in that PR or here as it is relevant to both.

**Review by hadifawaz1999 — APPROVED** [review]

**Comment by aiwalter** [comment]
Will review next week, on vacation this week.

**Review by TonyBagnall — DISMISSED** [review]

File: docs/get_involved/governance.md [hunk 14]
```diff
@@ -0,0 +1,151 @@
...
+on all project decisions. They are expected to review code contributions
+and engage with topics or code they are knowledgeable about.
+
+New core developers are nominated by existing core developers and are subject to a
+two-thirds majority vote of existing core developers. Core developers are expected to
+maintain a reasonable amount of engagement with the project. Core developers who no
+longer want or are able to engage with the project are expected to resign, but may lose
+their role after a long period of inactivity and lack of engagement with fellow
+developers if they neglect to do so. Developing code, interacting with contributions and
+engaging with the broader community are all valid contributions for core developers.
```
Comment (aiwalter) [inline docs/get_involved/governance.md:40]: I think this is quite informal e.g. what means "long period of time"?

File: docs/get_involved/governance.md [hunk 15]
```diff
@@ -0,0 +1,151 @@
...
+Twitter, LinkedIn, Slack and other relevant communications accounts.
+
+The communications team maintains access to social networking accounts and is
+responsible for managing access to the `aeon` email address. To help manage
+GitHub discussions, the communication workgroup is given triage access to the `aeon`
+GitHub repository.
+
+#### Code of Conduct Workgroup
+
+The Code of Conduct Workgroup (CoCW) consists of contributors tasked with making sure
```
Comment (aiwalter) [inline docs/get_involved/governance.md:91]: we might then have to rename the `COCC` on website as well?

File: docs/get_involved/governance.md [hunk 16]
```diff
@@ -0,0 +1,151 @@
...
+If a change is rejected, it is expected that some form of explanation and description
+of conditions (if any) to withdraw the rejection is provided.
+
+At any point during the discussion, any core developer in favour of a change can call
+for a vote, which will conclude two weeks from the call for the vote. Any vote must be
+backed by an AEP (see following section). In the event a vote is called, the proposal
+must receive a two-thirds majority of core developers to be approved.
+
+All changes to the `aeon` code or documentation should be done via Pull Request.
+By default, push rights to the `main` GitHub branch are restricted for all developers.
```
Comment (aiwalter) [inline docs/get_involved/governance.md:125]:
```suggestion
By default, push rights to the `main` GitHub branch are restricted for all core developers.
```

File: docs/get_involved/governance.md [hunk 17]
```diff
@@ -0,0 +1,151 @@
...
+In emergencies, the infrastructure workgroup may temporarily revoke the branch
+protection for group members and make direct commits, but this should happen only in
+emergencies where harm will come to the project unless timely action is taken.
+
+### Enhancement Proposals
+
+For all non-appointment decision-making votes, a proposal must have been made public
```
Comment (aiwalter) [inline docs/get_involved/governance.md:132]: I would add here the possibility to create a simple issue on GH to lower the barrier? Not every decision objective requires an AEP imo. E.g. adding a dependency e.g. ?

**Review by aiwalter — COMMENTED** [review]
left a few mostly minor comments. This means we will not have sth like a Community Council anymore, right? Which would be fine with me, but I think then we need to better define the deprecation process of inactive core developers

**Review by lmmentel — APPROVED** [review]

File: docs/get_involved/governance.md (hunk 15 above)
Comment (MatthewMiddlehurst) [reply docs/get_involved/governance.md:91]: True, I can make a follow-up for the governance page, code of conduct document, etc if this is approved.

File: docs/get_involved/governance.md (hunk 17 above)
Comment (MatthewMiddlehurst) [reply docs/get_involved/governance.md:132]: A definition for an "AEP" does not really exist, so it could be a big templated thing like sklearn does or just an issue giving some explanation. I can include a statement like that in the section if you want.

This would only be for situations where someone vetos a decision/PR, and instead of finding a consensus the one pushing the change decides to call a vote. Which from experience so far, seems to be a pretty rare occurrence.

File: docs/source/get_involved/governance.md (hunk 7 above)
Comment (MatthewMiddlehurst) [reply docs/source/get_involved/governance.md]: Given the new comments, is the length of votes still a change you think should happen?

File: docs/get_involved/governance.md (hunk 14 above)
Comment (MatthewMiddlehurst) [reply docs/get_involved/governance.md:40]: We can formalise it a bit more, but we would have to define A. in what situation would someone be removed (no contribution for x amount of time, no contact on any communication channels for x amount of time?), B. how long does someone have to be inactive to be removed, how much advance warning do they need to be given? C. who/what group has the responsibility to start this procedure and try to make contact with the inactive person

File: docs/source/get_involved/governance.md (hunk 7 above)
Comment (MatthewMiddlehurst) [reply docs/source/get_involved/governance.md]: An issue or PR as an altenrative to an AEP has been included in the enhancment proposal section if everyone involed agrees.

I have clarified when a AEP is needed (contentious PR votes where there is a rejection or major framework changes).

File: docs/get_involved/governance.md (hunk 14 above)
Comment (MatthewMiddlehurst) [reply docs/get_involved/governance.md]: I have formalised this a bit more deciding on some of the above.

As of the current document, after 1 year with no contributions (Developing code, interacting with contributions and engaging with the broader community), any core developer can make a PR to remove another core developer meeting this criteria. If this PR passes they are formally removed from the role.

File: docs/get_involved/governance.md (hunk 17 above)
Comment (MatthewMiddlehurst) [reply docs/get_involved/governance.md]: See above comments. I have clarified when a AEP is needed, what an AEP is with a template suggestion, and alternative methods of making a proposal.

For changes that no one rejects (even with rejections as long as there is no vote called), there is no need to make such a document.

**Review by TonyBagnall — APPROVED** [review]

**Review by hadifawaz1999 — APPROVED** [review]

**Review by dguijo — APPROVED** [review]

**Review by patrickzib — APPROVED** [review]

**Review by chrisholder — APPROVED** [review]

**Comment by MatthewMiddlehurst** [comment]
This currently has 5 approvals and one previous approval by @lmmentel. Could you respond to the comments on the open conversations @aiwalter when you find the time? No pressure to approve if you still have issues, but would be good to make progress.

**Comment by MatthewMiddlehurst** [comment]
I am calling a vote on this PR. It will conclude 16/10/2023. Core developers can either signal approval (i.e. '+1' or 'I approve') or rejection (i.e. -1 or 'I reject') in the posts below this. Going by current rules, a 2/3 majority is required to pass.

**Comment by MatthewMiddlehurst** [comment]
+1

**Comment by chrisholder** [comment]
+1

**Comment by TonyBagnall** [comment]
+1

**Comment by baraline** [comment]
+1

**Comment by dguijo** [comment]
+1

**Comment by hadifawaz1999** [comment]
+1

**Comment by patrickzib** [comment]
+1

**Comment by ltsaprounis** [comment]
+1

**Comment by MatthewMiddlehurst** [comment]
Vote has passed 8 Yes to 0 No. Merging.
